web_ui/frontend/components/Downtime/Registry/ServerUnknownDowntimeForm.tsx

```diff
@@ -71,48 +76,69 @@ const ServerUnknownDowntimeForm = ({
     'getNamespaces',
     getNamespaces
   );
-  const prefixes = useMemo(
-    () => namespacesToOriginAndCachePrefixes(namespaces || []),
-    [namespaces]
-  );
+  const servers = useMemo(() => {
```

Comment:
Don't process until data is actually loaded
```suggestion
  const servers = useMemo(() => {
  if (isLoading || !namespaces || namespaces.length === 0) {
      return [];
    }
```